tests/integration/src/rust_masm_tests/abi_transform/stdlib.rs

```diff
@@ -17,6 +17,7 @@ use proptest::{
 use crate::CompilerTest;
 
 #[test]
+#[ignore = "until builtin.global_symbol is implemented"]
```

Comment:
Should we create an issue for this?

Reply:
Nah, I'm already working on this one, and will have a PR open shortly